medusa/providers/torrent/html/hebits.py

```diff
@@ -0,0 +1,202 @@
+        self.url = 'https://hebits.net'
+        self.urls = {
+            'login': urljoin(self.url, 'takeloginAjax.php'),
+            'search': urljoin(self.url, 'browse.php'),
+        }
+
+        # Proper Strings
+        self.proper_strings = ['PROPER', 'REPACK', 'REAL', 'RERIP']
+
+        # Miscellaneous Options
+        self.freeleech = False
+
+        # Torrent Stats
+        self.minseed = None
+        self.minleech = None
+
+        # Cache
+        self.cache = tv.Cache(self)
+
+    def search(self, search_strings, age=0, ep_obj=None):
+        """
+        Search a provider and parse the results.
+
+        :param search_strings: A dict with mode (key) and the search value (value)
+        :param age: Not used
+        :param ep_obj: Not used
+        :returns: A list of search results (structure)
+        """
+        results = []
+        if not self.login():
+            return results
+
+        # Search Params
+        # Israeli SD , English SD , Israeli HD , English HD
+        # 7          , 24         , 37         , 1
+        search_params = {
+            'c1': '1',
+            'c24': '1',
+            'c37': '1',
+            'c7': '1',
+            'search': '',
+            'cata': '0',
+        }
+        if self.freeleech:
+            search_params['free'] = 'on'
+
+        for mode in search_strings:
+            log.debug('Search mode: {0}', mode)
+
+            for search_string in search_strings[mode]:
+
+                if mode != 'RSS':
+                    search_params['search'] = search_string
+                    log.debug('Search string: {search}',
+                              {'search': search_string})
+
+                response = self.session.get(self.urls['search'], params=search_params)
+                if not response or not response.text:
+                    log.debug('No data returned from provider')
+                    continue
+
+                results += self.parse(response.text, mode)
+
+        return results
+
+    def parse(self, data, mode):
+        """
+        Parse search results for items.
+
+        :param data: The raw response from a search
+        :param mode: The current mode used to search, e.g. RSS
+
+        :return: A list of items found
+        """
+        items = []
+
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find('div', class_='browse')
+            torrent_rows = torrent_table('div', class_=re.compile("^line")) if torrent_table else []
+
+            # Continue only if at least one release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return items
+
+            for row in torrent_rows:
+                try:
+                    heb_eng_title = row.find('div', class_='bTitle').find(href=re.compile("details\.php")).find('b').get_text(strip=True)
+                    title = heb_eng_title.split('/')[1].strip()
```

Comment:
Do you really need to strip() again here (the line above has already strip=True).

Reply:
This one is necessary. The first isn't so I deleted the strip=True.